.gitmodules

```diff
@@ -50,3 +50,7 @@
 [submodule "thirdparty/ittapi/ittapi"]
 	path = thirdparty/ittapi/ittapi
 	url = https://github.com/intel/ittapi.git
+[submodule "thirdparty/onednn_gpu"]
```

Comment:
@ilya-lavrenov What do you think on folders/submodules naming? Most of the internal folders use underscore, but several high level folders (inference-engine, model-optimizer, mkl-dnn, include/multi-device) use dash. Shouldn't it be aligned to one version?

Reply:
Sorry, missed the comment.

I personally ok with `_` names

Reply:
Is there a need to specify `_gpu`? Looks like the thirdparty can be used anywhere else besides the GPU

Reply:
We have another onednn in `inference-engine/thirdparty/mkl-dnn`, and I assume that in the future it will be moved to `<root>/thirdparty/onednn*`, thus we'll need to distinguish CPU and GPU versions of oneDNN. 

Anyway, there is no another user of this submodule so far. If we'll have any, submodule can be renamed in the future.